routes/common/postgres_helper.rb

```diff
@@ -64,12 +64,8 @@ def get
   def delete
     Authorization.authorize(@user.id, "Postgres:delete", @resource.id)
     @resource.incr_destroy
-    if @mode == AppMode::API
-      response.status = 204
-      @request.halt
-    else
-      {message: "Deleting #{@resource.name}"}.to_json
-    end
+    response.status = 204
```

Comment:
I assume that the ajax request in the UI assumes 204 as success. But it worths to check.

Reply:
Yes it does.